Gemfile

```
@@ -9,4 +9,4 @@ gem "without_steep_types", path: "test/gems/without_steep_types"
 gem "rake"
 gem "minitest", "~> 5.15"
 gem "minitest-hooks"
-gem "stackprof"
+gem "stackprof" unless Gem.win_platform?
```

We may want to use optional group here.

```suggestion
group :stackprof, optional: true do
  gem "stackprof"
end
```

So, we can install `bundle install --with stackprof` when we need it.